packages/dev/core/src/Gizmos/gizmo.ts

```diff
@@ -332,6 +332,7 @@ export class Gizmo implements IGizmo {
                     effectiveNode.getClassName() === "InstancedMesh";
                 const transformNode = supportedNode ? (effectiveNode as TransformNode) : undefined;
                 effectiveNode.getWorldMatrix().decompose(undefined, this._rootMesh.rotationQuaternion!, undefined, Gizmo.PreserveScaling ? transformNode : undefined);
+                this._rootMesh.rotationQuaternion?.normalize();

```

Comment:
if you are sure it is defined use `!` instead of `?`, because `?` will generate unneeded code:

```suggestion
                this._rootMesh.rotationQuaternion!.normalize();
```